[APPSEC-48] Modify rule S2083[java]: Educational content (#1112)
SonarSource/rspec: rules/S2083/rationale.adoc

Path injections occur when an application uses untrusted data to construct a file path and access this file without validating its path first.

> A user with malicious intents would inject specially crafted values, such as ``++../++``, to change the initial path and, when accessed, resolve to a path on the filesystem where the user should normally not have access.

Reviewer: ```suggestion
A user with malicious intent would inject specially crafted values, such as ``++../++``, to change the initial intended path. The resulting path would resolve somewhere in the filesystem where the user should not normally have access to.
```
